beacon-chain/rpc/service.go

```diff
@@ -301,24 +323,107 @@ func (s *Service) ValidatorIndex(ctx context.Context, req *pb.PublicKey) (*pb.In
 	return &pb.IndexResponse{Index: index}, nil
 }
 
-// LatestAttestation streams the latest processed attestations to the rpc clients.
-func (s *Service) LatestAttestation(req *empty.Empty, stream pb.BeaconService_LatestAttestationServer) error {
-	sub := s.attestationService.IncomingAttestationFeed().Subscribe(s.incomingAttestation)
-	defer sub.Unsubscribe()
+// ValidatorAssignment streams validator assignments every slot to clients that request
+// to watch a subset of public keys in the CrystallizedState's active validator set.
+func (s *Service) ValidatorAssignment(req *pb.ValidatorAssignmentRequest, stream pb.ValidatorService_ValidatorAssignmentServer) error {
+	// If the genesis time was at 12:00:00PM a if the SlotDuration is set to
```

Comment:
This comment seems to be repeated several times. Shouldn't it be part of the godoc and the interested reader can look there for more info?